pycbc/workflow/pegasus_workflow.py

```diff
@@ -112,6 +112,30 @@ def add_profile(self, namespace, key, value, force=False):
                 self._dax_executable.removeProfile(entry)
                 self._dax_executable.addProfile(entry)
 
+    def is_same_as(self, other):
+        test_vals = ['namespace', 'version', 'arch', 'os', 'osrelease',
+                     'osversion', 'glibc', 'installed', 'container']
+        # Check for logical name first
+        if not self.pegasus_name == other.pegasus_name:
```

Comment:
So another check would be needed here the tagged name were removed in the case another executable was desired.